### ISSUE TITLE
Include EWAH

### ISSUE THREAD

**Issue by lemire** [BitSetsNet/BitSetsNet#14]
Great project...

You might be able to include EWAH in your library...

https://github.com/lemire/csharpewah

**Comment by oahassan** [comment]
@lemire Definitely. Is it OK to wrap your implementation with our IBitSet interface?

**Comment by sgf** [comment]
> Great project...
> 
> You might be able to include EWAH in your library...
> 
> https://github.com/lemire/csharpewah

Does EWAH have any advantages over Roaring?
